<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>runplandialog</class>
 <widget class="QDialog" name="runplandialog">
  <property name="windowTitle">
   <string>跑步计划参数</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>体重 (kg):</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDoubleSpinBox" name="weightSpinBox">
     <property name="minimum">
      <double>30.0000000000</double>
     </property>
     <property name="maximum">
      <double>200.0000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>身高 (cm):</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="heightSpinBox">
     <property name="minimum">
      <double>100.0000000000</double>
     </property>
     <property name="maximum">
      <double>250.0000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>剩余里程 (km):</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="kmDoubleSpinBox">
     <property name="minimum">
      <double>0.1000000000</double>
     </property>
     <property name="maximum">
      <double>1000.0000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>单次最长跑步 (分钟):</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="maxTimeSpinBox">
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>240</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>目标完成日期:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDateEdit" name="dateEdit">
     <property name="minimumDate">
      <date>2000-01-01</date>
     </property>
     <property name="maximumDate">
      <date>2100-12-31</date>
     </property>
     <property name="date">
      <date>2024-03-01</date>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>runplandialog</receiver>
   <slot>accept()</slot>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>runplandialog</receiver>
   <slot>reject()</slot>
  </connection>
 </connections>
</ui>
